service-deviceregistry/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# pull official base image
+FROM python:3.11.3-slim
+
+# create directory for the app user
+RUN mkdir -p /project
+
+# create the app user
+RUN addgroup --system app && adduser --system --group app
+
+ENV APP_HOME=/home/app
+WORKDIR $APP_HOME
+
+# copy project
+COPY . $APP_HOME
+
+# install dependencies
+COPY requirements.txt .
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir -r requirements.txt
+
+# chown all the files to the app user
+RUN chown -R app:app $APP_HOME
+
+#give permission to store media
+RUN chown -R app:app /media
+
+# change to the app user
+USER app
+
+ENV DJANGO_SETTINGS_MODULE=deviceregistry.settings
+# start django server
+CMD python manage.py runserver 0.0.0.0:8000
```

Comment:
Before putting anything into staging or production, I think we should use gunicorn (or what is state-of-the-art wsgi process nowadays?) and a proxy (nginx?) Also we probably want to use the same `DEBUG=False` in settings.py  in staging and production and other deployment related variables (ALLOWED_HOSTS etc) defined in envs? @laurigates @sheenacodes

Reply:
Since this is not production ready yet, can we get all services (endpoint and parser) to this repo and to run on docker-compose env for now as first step ? We can work on production related changes after that ?